Userland/Libraries/LibMarkdown/ContainerBlock.cpp

```diff
@@ -119,16 +129,18 @@ OwnPtr<ContainerBlock> ContainerBlock::parse(LineIterator& lines)
             has_blank_lines = has_blank_lines || has_trailing_blank_lines;
         }
 
+        bool is_interrupting_paragraph = not paragraph_text.is_empty();
```

Comment:
That's curious. We tend not to use the keyword versions of operators though, so this should probably be !paragraph_text.is_empty()